decisions.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"math/rand"
+	"strings"
+)
+
+func choose(nick string, msg string) string {

```

Comment:
```suggestion
func choose(nick, msg string) string {
```

Reply:
Addressed in https://github.com/bytebot-chat/bytebot-party-pack/pull/26/commits/16f9f02a1310ad43293a85c74411f8111ae72d2a